src/components/ActionableInput/index.tsx

```diff
@@ -59,17 +61,32 @@ const ButtonContainer = styled(Button)<{ $wasClicked?: boolean }>`
   }}
 `
 
+const ShowButton = styled(Button)`
+  margin-left: -74px;
+  z-index: 2;
+`
+
+const RotateKeyButton = styled(Button)`
+  margin-left: 55px;
+  width: 120;
+  img {
+    padding-right: 10px;
+  }
+`
+
 export type ActionableInputProps = InputProps & {
   onClickCopy?: (value: string) => void
   shouldShowHideShowButton?: boolean
   testId?: string
+  onClickRotateModal?: (value: boolean) => void
```

Comment:
components should be general and extensible, rather than hard-coding specific use-cases. it would be better to allow this component to accept custom buttons

Reply:
Upon running this change locally, I don't see why this needs to be part of actionable input at all